ansible/roles/kafka-logging-setup/tasks/main.yml

```diff
@@ -1,15 +1,42 @@
 ---
-- name: Deploy Kafka cluster
+- name: Label Kafka nodes
+  k8s:
+    state: present
+      definition: "{{ lookup('template', 'node.yml.j2') }}"
```

Comment:
I haven't tested if this works but I think `definition` is indented too much and should be at the same indentation as `state`.

Reply:
fixed